util/topsql/reporter/reporter.go

```diff
@@ -155,6 +169,7 @@ func (tsr *RemoteTopSQLReporter) RegisterSQL(sqlDigest []byte, normalizedSQL str
 // This function is thread-safe and efficient.
 func (tsr *RemoteTopSQLReporter) RegisterPlan(planDigest []byte, normalizedBinaryPlan string) {
 	if tsr.planMapLength.Load() >= variable.TopSQLVariable.MaxCollect.Load() {
+		ignoreRegisterPlanCnt.Inc()
```

Comment:
maybe better to distinguish it as "ignore_exceed".

Reply:
great, done.